torch/_inductor/ops_handler.py

```diff
@@ -712,3 +722,42 @@ def constant(self, value: Any, dtype: torch.dtype) -> "torch._inductor.ir.Consta
 
 def _typecheck_ExtractConstantsHandler(h: ExtractConstantsHandler) -> OpsHandler[Any]:
     return h
+
+
+class SimpleCSEHandler(WrapperHandler[T]):
```

Comment:
Why do you need this instead of the normal CSE handler?

Reply:
So this isn't strictly necessary. We would just toss the unfiltered `inner_fn`s into a closure and leave it for codegen to clean up.

I was thinking at some point we might want a real fx graph so we could do pattern matching or value range analysis or what have you, but not at the moment.